db/warehouse/migrate/20231004203202_add_email_index_to_hud_user.rb

```diff
@@ -0,0 +1,8 @@
+class AddEmailIndexToHudUser < ActiveRecord::Migration[6.1]
+  def change
+    StrongMigrations.disable_check(:add_index)
+    add_index :User, [:data_source_id, :UserEmail], unique: false, name: :ds_email_idx
```

Comment:
This should be 
```suggestion
    add_index :User, [:UserEmail, :data_source_id], unique: false, name: :users_ds_email_idx
```
You want the more distributed item first, and index names are unique across the DB, so we should include the table name.

Reply:
Thanks!